srcpkgs/nagios/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/$pkgname/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+	--with-nagios-user=$_nagios_user
+	--with-nagios-group=$_nagios_group
+	--prefix=$_instdir
+	--bindir=$_bindir
+	--localstatedir=$_vardir
+	--sysconfdir=$_confdir
+	--with-httpd-conf=$_httpdconfdir
+	--with-checkresultdir=$_checkresultdir
+	--with-cgibindir=$_cgibindir
+	--enable-embedded-perl"
+make_build_args="all"
+make_install_args="
+	prefix=$_instdir
+	BINDIR=$_bindir
+	LOGDIR=$_vardir
+	CFGDIR=$_confdir
+	HTTPD_CONF=$_httpdconfdir
+	CHECKRESULTDIR=$_checkresultdir
+	CGIDIR=$_cgibindir
+	install install-config"
+
+makedepends="perl unzip"
+make_dirs="/etc/nagios 0775 _nagios _nagios"
+conf_files="/etc/nagios/nagios.cfg"
+
+short_desc="Nagios Core"
+maintainer="Dave Eddy <dave@daveeddy.com>"
+license="GPL-2.0-or-later"
+
+homepage="https://github.com/NagiosEnterprises/nagioscore"
+changelog="${homepage}/blob/master/Changelog"
+distfiles="${homepage}/releases/download/${pkgname}-${version}/${pkgname}-${version}.tar.gz"
+
+checksum=ab0d5a52caf01e6f4dcd84252c4eb5df5a24f90bb7f951f03875eef54f5ab0f4
+system_accounts="${_nagios_user}"
+
+post_install() {
+	# these dirs are wiped away by the void post install hooks since they are empty
+	vmkdir "$_vardir/rw"
+	vmkdir "$_vardir/spool/check/results"
```

Comment:
If these directories are deleted, why are you adding them? You can specify directories to be created with `make_dirs`. Also, `vmkdir` works with `usr/share`, not `/usr/share`.

Reply:
hmm you have `checkresults` above and `check/results` here. Is that correct?

Reply:
The directories are deleted by the void post install hooks (out of the control of this template AFAICT).  They are required by nagios however, even though they are empty.  I added this logic to just recreate them after they were removed by the hooks provided by this repository.

I'll switch this over to use `make_dirs`

---

Yes! good catch, that's my mistake.